mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/MavenPropertyResolver.java

```diff
@@ -18,4 +24,14 @@ static private boolean isSkip() {
         return Boolean.valueOf(System.getProperty("maven.test.skip", "false"));
     }
 
+    static boolean isSpecificTestClassSet() {
+        String specificTestClasses = System.getProperty("test");
+        return specificTestClasses != null && !containsPattern(specificTestClasses);
+    }
+
+    private static boolean containsPattern(String specificTestClasses) {
+        return Arrays.stream(specificTestClasses.split(","))
+            .map(TEST_CLASS_PATTERN::matcher)
+            .anyMatch(Matcher::find);
```

Comment:
You shouldn't check if there is at least one specific test specified. You should check if there is some part containing a pattern. 
eg:
`-Dtest=SimpleOneTest,Multiple*Test`
or am I missing something?
The test could show more

Reply:
I don't get this one. Here, I am splitting the list of test classes set with `test` property and traversing over each test to check if any of it contains pattern.  Probably the variable`specificTestClasses` is confusing. 
Will fix that and add a few more tests for all the cases.  I had added few generic test cases because in my opinion the functionality was more or less the same.

Reply:
> Will fix that and add a few more tests for all the cases

Did we ever fix that?

Reply:
@hemanik Yeah, you are right - this is correct - I misread it.